zebra/zebra_vxlan.c

```diff
@@ -3006,18 +3007,20 @@ int zebra_vxlan_clear_dup_detect_vni_mac(struct zebra_vrf *zvrf, vni_t vni,
 
 	zevpn = zebra_evpn_lookup(vni);
 	if (!zevpn) {
-		zlog_warn("VNI %u does not exist\n", vni);
+		snprintf(errmsg, errmsg_len, "VNI %u does not exist", vni);
```

Comment:
Minor nit: Please use snprintfrr instead. https://github.com/FRRouting/frr/pull/6855 is trying to replace all old snprintf.

Reply:
I personally think using `snprintf()` is fine here. I usually use `snprintfrr()` only when necessary (e.g. when I need to use a FRR-specific format specifier).